### ISSUE TITLE
IE11: Video shows a black screen when screen-reader is present

### ISSUE THREAD

**Issue by maketroli** [videojs/video.js#5071]
## Description
The video is working properly on every browser but I need it to be accessible.
Every time I activate an screen reader named JAWS the video shows a black screen.

http://jsbin.com/axedog/edit?html,output

## Steps to reproduce
Explain in detail the exact steps necessary to reproduce the issue.
1. Open browser (IE11) with the proper page containing the video
2. Open JAWS (the screen reader)
3. You will see a black screen in the video.

### Expected
The video should be present, accessible and showing correctly.

### Actual
Video shows a black screen.

## Additional Information
Please include any additional information necessary here. Including the following:
#### videojs
Latest
#### browsers
IE11
#### OSes
Windows

**Comment by gkatsev** [comment]
What version of JAWS are you using? Also, which version of windows?
We've been tested JAWS and Video.js and IE11, at least on windows 10 on a semi-regular basis and it's been working fine.

**Comment by maketroli** [comment]
JAWS 17 and Windows 7

**Comment by OwenEdwards** [comment]
The only way I can reproduce this behavior on IE 11 with JAWS 17 on Windows 7 is by noting that the example at http://jsbin.com/axedog/edit?html,output is somewhat outdated (it uses video.js v5.0, which is deprecated) but also that it uses **http** links to the video.js CSS and JS files, but JS Bin forces the URL to use **https**, so IE complains about this by showing an "Only secure content is displayed" warning:

![2018-11-05_16-57-35](https://user-images.githubusercontent.com/2245205/48036409-18224380-e11d-11e8-84d3-8ef83d684d37.png)

If you click "Show all content" (using JAWS), then the usual video.js Big Play Button shows up on the page, and you can play the video as expected.

So @gkatsev, I think the only issue here is that the JS Bin example at http://jsbin.com/axedog/edit?html,output needs updating to use **https**, and to use a version of video.js which is still supported (v6.x or v7.x).

**Comment by OwenEdwards** [comment]
See #5562.
